### PR TITLE
Solve some todos

### PR DESCRIPTION
**Pull request by Flova**
This is related to issue #92.
@NFiedler or @hendrikbrandt could one of you add the missing documentation for the parameters in the `vision.cfg`?

### CODE REVIEW

File: bitbots_vision/cfg/Vision.cfg [hunk 1]
```diff
@@ -152,7 +152,7 @@ group_field_boundary_detector.add("field_boundary_detector_vertical_steps", int_
 group_field_boundary_detector.add("field_boundary_detector_horizontal_steps", int_t, 0, "Number of scanlines", min=1, max=640)
 group_field_boundary_detector.add("field_boundary_detector_roi_height", int_t, 0, "Region Of Interest height in which we are looking for green", min=1, max=100)
 group_field_boundary_detector.add("field_boundary_detector_roi_width", int_t, 0, "Region Of Interest width in which we are looking for green", min=1, max=100)
-group_field_boundary_detector.add("field_boundary_detector_roi_increase", double_t, 0, "Value that increases the if it gets lower in the image", min=0, max=1.0) # TODO: increases WHAT if WHAT...?
+group_field_boundary_detector.add("field_boundary_detector_roi_increase", double_t, 0, "Value that increases the region of interest the if it is located lower in the image", min=0, max=1.0)
```
Comment (JanGut) [inline bitbots_vision/cfg/Vision.cfg]:
```suggestion
group_field_boundary_detector.add("field_boundary_detector_roi_increase", double_t, 0, "Value that increases the region of interest if it is located lower in the image", min=0, max=1.0)
```

File: bitbots_vision/src/bitbots_vision/vision_modules/lines.py [hunk 2]
```diff
@@ -152,7 +152,6 @@ def filter_points_with_candidates(linepoints, candidates):
         """
         filtered_linepoints = []
         for linepoint in linepoints:
-            # TODO: use list comprehension instead of map
```
Comment (JanGut) [inline bitbots_vision/src/bitbots_vision/vision_modules/lines.py]: This has been done in 97d974528a60dae41e4f7604de14d875c42fa10b.
Thanks for removing the comment.

**Comment by JanGut** [comment]
Please check if the change to _underscore at `_conventional_modules` has been merged correctly.

File: .gitignore [hunk 3]
```diff
@@ -195,3 +195,9 @@ ENV/
 .ropeproject
 
 models/
+
+
+# auto-generated documentation
+**/docs/_build
+**/docs/_out
+**/docs/conf.py
```
Comment (Flova) [inline .gitignore]: Sry this was added by accident.

**Comment by JanGut** [comment]
A kind reminder:

> @NFiedler or @hendrikbrandt could one of you add the missing documentation for the parameters in the vision.cfg?

**Pull request merged**